Rename the gem to rails-bigint-primarykey

rafaelfranca opened the pull request:
This will make possible to us to release it in rubygems.org and it will make explicit this is a rewrite of the original gem.

After merging this I'll rename the repository and ask GitHub support to remove the fork link.

lugray commented on lib/bigint_primarykey.rb:
```diff
@@ -85,7 +85,7 @@ def install_patches!(adapter)
         pk_module = MysqlBigintPrimaryKey
```
Not strictly necessary, but I'd `s/pk_module/primary_key_module/` here and L81,L95

lugray commented:
Another time, it would be helpful for review to have the rubocop applied changes in a separate commit.

rafaelfranca commented:
> Another time, it would be helpful for review to have the rubocop applied changes in a separate commit.

Yup. Totally agree. Let me fix this.

rafaelfranca committed:
Rename the gem to rails-bigint-primarykey

This will make possible to us to release it in rubygems.org and it will
make explict this is a rewrite of the original gem.

rafaelfranca commented:
Cleaned the changes to not include the rubocop fixes. Also fixed the comment

lugray commented on test/gemfiles/Gemfile-Rails-4.2:
```diff
@@ -1,6 +1,6 @@
 source "https://rubygems.org"
 
-# Specify your gem's dependencies in rails-bigint-pk.gemspec
+# Specify your gem's dependencies in rails-bigint-primarykey.gemspec
```
This change is needed in `test/gemfiles/Gemfile-Rails-5.0` too.

Edouard-chin commented on Readme.md:
````diff
@@ -23,3 +29,6 @@ end
 ```
+
+When upgrading to Rails 5.1 you still need this gem if your were using it before and your migrations still use the
+verion 5.0 or previous.
````
`verion` -> `version`

Edouard-chin commented:
Educate me here, should a fork have it's own versioning ? Just curious about the `2.0.0` for the first version of a fork

rafaelfranca commented:
> Educate me here, should a fork have it's own versioning?

It can, but in this case I prefer to keep the version schema the same as the original one. It is fine to jump versions, the first version of actioncable for is the 5.0.0.
